lib/Backends/OpenCL/OpenCL.h

```diff
@@ -96,7 +96,7 @@ class OpenCLFunction final : public CompiledFunction {
   ///@{
   ~OpenCLFunction() override;
 
-  void execute() override;
+  void execute(Context *ctx) override;
   ///@}
```

Comment:
not related to PR but ///@? needs to be closed after `void tearDownRuns() override;`